updater/aws_test.go

```diff
@@ -15,6 +15,61 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestGetCommandResult(t *testing.T) {
+	cases := []struct {
+		name          string
+		invocationOut *ssm.GetCommandInvocationOutput
+		expectedError string
+		expectedOut   []byte
+		getError      error
```

Comment:
`invocationError` would be a better name corresponding to `invocationOut`
```suggestion
		invocationError      error
```